app/pages/home-not-logged-in.cjsx

```diff
@@ -32,30 +32,29 @@ module.exports = React.createClass
             <source type="video/mp4" src="./assets/home-video.mp4"></source>
           </video>
         }
-
         <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
-        <h2 className="lighter">is people powered research</h2>
-        <h2>
-          <Link to="/about" className="standard-button">Learn More</Link>
-        </h2>
+        <h2 className="lighter">is people powered-research</h2>
```

Comment:
"powered-research" shouldn't be hyphenated.  Alex mentioned hyphenating "people powered" , but that's optional grammar.  (Personally, I wouldn't)